src/components/NavBar.jsx

```diff
@@ -1,9 +1,9 @@
 const NavBar = () => {
   return (
-    <span>
-      <button id="cat_button">Category 1</button>
-      <button id="cat_button">Category 2</button>
-      <button id="cat_button">Category 3</button>
+    <span className="container nav">
```

Comment:
I'm sure this is a generic placeholder for now. 
But think about how you'd like this to appear on a smaller screen - you may have much more than 3 categories, which would appear much too crowded